src/freedesktop.rs

```diff
@@ -213,6 +213,16 @@ pub fn list() -> Result<Vec<TrashItem>, Error> {
     Ok(result)
 }
 
+/// The path points to:
+/// - existing file | directory | symlink => Ok(true)
+/// - broken symlink => Ok(true)
+/// - nothing => Ok(false)
+/// - I/O Error => Err(Io)
+#[inline]
+pub fn virtually_exists(path: &Path) -> std::io::Result<bool> {
```

Comment:
I think I would keep the amount of API that is platform-specific to a minimum, and since it's only used in this module it can and should be private.

Reply:
Done.